src/core/retryer.ts

```diff
@@ -6,6 +6,7 @@ import { sleep } from './utils'
 
 interface RetryerConfig<TData = unknown, TError = unknown> {
   fn: () => TData | Promise<TData>
+  onQuery?: () => void
```

Comment:
Would it be useful to pass query time variables as an argument to callback function?